README.md

```diff
@@ -21,19 +21,32 @@ Trix is built with established web standards, notably [Custom Elements](https://
 
 # Getting Started
 
-Include the bundled `trix.css` and `trix.js` files in the `<head>` of your page.
+Trix comes bundled in ESM and UMD formats and works with any asset packaging system.
+
+The easiest way to start with Trix is including it from an npm CDN in the `<head>` of your page:
```

Comment:
Maybe worth adding something like "...and then calling `Trix.start()` to initialise the library"?

The example snippet makes it pretty clear, but if it's a required step it might be worth calling it out just to be sure folks don't miss it.

Reply:
Good point, added in ea677ee90e7464d02639ae18f9204b69d7daeb57.